### ISSUE TITLE
Why I can't reproduce the result reported in the paper?

### ISSUE THREAD

**Issue by lgzlIlIlI** [MengyuanChen21/ECCV2022-DELU#4]
Hi Authors,
Thanks for your wonderful work！
I create a clean environment and follow the provided code to train the model, but the results on THUMOS14 are much lower than those reported in the paper.
Iteration: 4950, Loss: 2.23683
Testing test data point 0 of 212
Testing test data point 42 of 212
Testing test data point 84 of 212
Testing test data point 126 of 212
Testing test data point 168 of 212
Testing test data point 210 of 212
	Number of ground truth instances: 3358
	Number of predictions: 36474
Classification map 93.916390
map @ 0.1 = 67.505 ||map @ 0.2 = 62.470 ||map @ 0.3 = 53.532 ||map @ 0.4 = 45.423 ||map @ 0.5 = 37.230 ||map @ 0.6 = 26.003 ||map @ 0.7 = 13.898 ||map @ 0.8 = 7.284 ||map @ 0.9 = 2.063 
mAP Avg ALL: 35.045
MAX map @ 0.1 = 69.091 ||MAX map @ 0.2 = 63.765 ||MAX map @ 0.3 = 54.554 ||MAX map @ 0.4 = 45.960 ||MAX map @ 0.5 = 38.204 ||MAX map @ 0.6 = 26.796 ||MAX map @ 0.7 = 14.790 ||MAX map @ 0.8 = 7.460 ||MAX map @ 0.9 = 2.137 
mAP Avg 0.1-0.5: 54.3147951833503, mAP Avg 0.1-0.7: 44.73712870057078, mAP Avg ALL: 35.8617997448392
------------------pid: 1102141--------------------
I have the same configuration for the environment as the suggested. Can you give me some advice?

**Comment by MengyuanChen21** [comment]
I am really confused about the results. I ran the codes with random seeds 0 to 4, and obtained the results of 46.41, 46.30, 45.09, 45.84, and 45.88 (mAP 0.1-0.7). The lowest one is still higher than the one you reproduced.

Could you please try more random seeds on your machine and inform me of the results? I really want to figure out how the low results come out, and I sincerely expect you guys to reproduce my results.

**Comment by lgzlIlIlI** [comment]
Thanks for your reply. I have reproduced a result close to the original paper, and the environment I am currently using is TITAN xp, pytorch 1.11.0, CUDA 10.2. The reproduction results are as follows：

Iteration: 8800, Loss: 1.88242
map @ 0.1 = 68.351 ||map @ 0.2 = 62.972 ||map @ 0.3 = 54.797 ||map @ 0.4 = 46.432 ||map @ 0.5 = 38.831 ||map @ 0.6 = 27.133 ||map @ 0.7 = 14.622 ||map @ 0.8 = 7.571 ||map @ 0.9 = 1.836 
mAP Avg ALL: 35.838
MAX map @ 0.1 = 70.609 ||MAX map @ 0.2 = 65.181 ||MAX map @ 0.3 = 56.844 ||MAX map @ 0.4 = 47.770 ||MAX map @ 0.5 = 40.118 ||MAX map @ 0.6 = 27.610 ||MAX map @ 0.7 = 14.828 ||MAX map @ 0.8 = 7.479 ||MAX map @ 0.9 = 1.770 
mAP Avg 0.1-0.7: 46.13714796308268, mAP Avg 0.3-0.7: 37.43405233662147, mAP Avg ALL: 36.912144453575806

However，in Nvidia RTX 3090 GPU，when using a random seed of 0, the resulting avg mAP (0.1:0.7) result is 44.73%. I will continue to try with other random seeds on 3090 GPU. Besides, since the server I'm using is offline, I'm not using Wandb, will this make a difference? The environment used on the 3090 GPU is torch 1.11.0, CUDA 11.3

**Comment by MengyuanChen21** [comment]
Glad to hear about the new reproduced result.

I have ran my codes on both Nvidia RTX 3090 GPU and Nvidia TITAN RTX, the environment is torch 1.11, CUDA 11.1, and the results are similar, whose average over different random seeds is about 46%, and the best performance can reach 46.4%.
I have not tested the codes with different versions of PyTorch or CUDA, so I am also not very clear about the exact influence. But I guess there shouldn't be big problems with close versions.

Last night another researcher (who raised the data link issue on Dec 21, 2022) told me the results she/he reproduced is 46.116% with 5000 iterations. Therefore, I guess the 44.73% result is just a coincidence related to randomness (I hope so).

No, Wandb shouldn't affect the results, since it is only a visualization tool. Just ignore it if your server is offline.
